test/openAPI/features/callback_email.feature

```diff
@@ -4,34 +4,35 @@ Feature: Respond to an email sent via Messaging BB
     @smoke @unit @positive
     Scenario: Callback endpoint successfully responds to an email sent via Messaging BB smoke type test
 
-        Given Single email with given "abcdef12345" as api_key and valid payload is sent and returns requestUID
-        When "POST" request with given "abcdef12345" as api_key and valid requestUID in the headers is sent
+        Given Single email with given "abcdef12345" as api_key and valid payload is sent and returns requestUID, for example "btpn-aiqu-urw8-94zm-h0av-vjhl"
+        When "POST" request with given "abcdef12345" as api_key and valid requestUID, for example "btpn-aiqu-urw8-94zm-h0av-vjhl", in the headers is sent
```

Comment:
I don't think we should use "for example" in tests. Better option would be something more concise like: 
```gherkin
Given a single email with api_key "abcdef12345" and a valid payload is sent, returning a requestUID "btpn-aiqu-urw8-94zm-h0av-vjhl"
When a "POST" request with api_key "abcdef12345" and requestUID "btpn-aiqu-urw8-94zm-h0av-vjhl" in the headers is sent
```


This comment refeer also to other scenarios new steps with same syntax.